src/components/login.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Login = styled.div`
+    margin: 20px;
+    padding: 20px;
+    h2 {
+        font-weight: bold;
+        color: #4A494A;
+    }
+    form {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        div {
+            margin: 5px;
+        }
+    }
+`;
+
+class login extends Component {
+    state = {

```

Comment:
Love how you are using class fields to avoid having to bind your methods and / or using a constructor! 
If you'd like to refactor this to hooks, let me know 👍